scripts/linux_share_build.sh

```diff
@@ -1,13 +1,25 @@
 #!/bin/bash
+set -ue
+
+SUDO=""
```

Comment:
this should be moved in the `scripts/build_plugin_linux.sh` 
otherwise It overwrites the variable for CI